sympy/physics/_biomechanics/activation.py

```diff
@@ -0,0 +1,431 @@
+"""Activation dynamics for musclotendon models.
+
+Musculotendon models are able to produce active force when they are activated,
+which is when a chemical process has taken place within the muscle fibers
+causing them to voluntarily contract. Biologically this chemical process (the
+diffusion of $Ca^{2+}$ ions) is not the input in the system, electrical signals
```

Comment:
```suggestion
diffusion of :math:`Ca^{2+}` ions) is not the input in the system, electrical signals
```

I'm not sure, but I think all docstrings in py files need to be restructuredtext.

Reply:
Otherwise tools that aren't sphinx with the specific plugins we have installed can't parse them (e.g. an IDE's parser).